### PR TITLE
Opens a new viro slot

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
There can now be two virologists. The maps are all changed to compensate, with two separate virology labs that have doors boltable from the inside. Viral Base is now a virologist starting item, and doesn't come in chemfridges. Syndiebase still has viral base, and also got two cans of laughter for anomalous virus food. Finally, virologist heirloom is changed from a syringe to a dropper.

## Why It's Good For The Game
i made viro too fun and now i can barely play it myself reeee

## Changelog
:cl:
add: Syndiebase has a bottle of viral base and two cans of laughter
tweak:  Virologists now spawn with viral base in their right pocket, and it no longer appears in the viro fridge
tweak: virologist heirloom is now a dropper
tweak: virology has been remapped on most maps to now have two separate isolated labs that can be bolted from inside, perfect for two antisocial virologists
config: there are now two virologist slots
/:cl:

### CODE REVIEW

File: config/jobs.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ Medical Doctor=7,4
 Paramedic=2,1
 Chemist=3,2
 Geneticist=3,2
-Virologist=1,1
+Virologist=2,2
```
Comment (qwertyquerty) [inline config/jobs.txt:37]: Honestly I would make this 2,1

Comment (zeskorion) [reply config/jobs.txt:37]: I added new spawn positions as well

Comment (qwertyquerty) [reply config/jobs.txt:37]: It doesn't matter. I don't want two viros to spawn by default, but rather let another one join if they want to.

Comment (zeskorion) [reply config/jobs.txt:37]: that's... quite literally what this is for though. Latejoining places you at a disadvantage.

**Comment by qwertyquerty** [comment]
@Crossedfall figgys

**Pull request merged**